wandb/apis/public/runs.py

```diff
@@ -754,10 +754,54 @@ def scan_history(self, keys=None, page_size=1000, min_step=None, max_step=None):
 
     @normalize_exceptions
     def logged_artifacts(self, per_page=100):
+        """Return an iterable collection of all artifacts logged by this run.
+
+        This method retrieves all output artifacts that were logged during the run.
+
+        Arguments:
+            per_page (int, optional): Number of artifacts to fetch per API request. Defaults to 100.
+
+        Returns:
+            RunArtifacts: An iterable collection of all Artifact objects logged as outputs during this run.
+
+        Example:
+            ```python
+            run = api.run("entity/project/run_id")
+            for artifact in run.logged_artifacts():
+                print(f"{artifact.name}:{artifact.version}")
+            ```
+
+        Note:
+            This method returns a paginated result. You can iterate over all artifacts
+            or use list() to collect them into a single list.
```

Comment:
we follow the google docstring style

Reply:
https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings

Reply:
Noted. Unsure what's the action item here though.

Reply:
same as above using Args (linked the guide for more details)

Reply:
Changed the docstrings to match Google style.